ovos_dinkum_listener/service.py

```diff
@@ -508,12 +506,7 @@ def _hotword_audio(self, audio_bytes: bytes, ww_context: dict):
 
             if sound:
                 LOG.debug(f"Handling listen sound: {sound}")
-                try:
-                    sound = resolve_resource_file(sound, config=self.config)
-                    if sound:
-                        play_audio(sound)
-                except Exception as e:
-                    LOG.warning(e)
+                self.bus.emit(Message("mycroft.audio.play_sound", {"uri": sound}))
```

Comment:
Should this include some message context (routing, ww context?)

Reply:
this is the source message, just like the utterance message, it is not an answer to anything, but should probably include context for source and destination

ovos-audio side should probably ignore these messages like it does TTS etc for non-native sources